### PR TITLE
refactor(examples): Static 

### PR DESCRIPTION
**Pull request by justyna-13**
# Scope of work
Zrzut ekranu z 2020-03-24 16-21-33

### CODE REVIEW

File: packages/vue/src/examples/pages/static/Static.vue [hunk 1]
```diff
@@ -100,12 +135,19 @@ export default {
...
   }
 }
+
 .breadcrumbs {
-  padding: var(--spacer-big) var(--spacer-extra-big) var(--spacer-extra-big)
-    var(--spacer-extra-big);
+  margin: var(--spacer-base) auto var(--spacer-lg);
+}
+
+.title {
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/static/Static.vue:147]: maybe you can use SfHeading level 3, then you can remove this class

File: packages/vue/src/examples/pages/static/Static.vue [hunk 2]
```diff
@@ -60,18 +72,46 @@
         information collected offline by Luma. All Luma websites are covered by
         this privacy policy.
       </SfContentPage>
+      <SfContentPage title="Delivery">
+        This website ("website") is operated by Luma Inc., which includes Luma
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/static/Static.vue:76]: Zrzut ekranu 2020-04-1 o 14 11 59
maybe we can add padding: 0 var(--spacer-sm) ?

**Comment by pspaczek** [comment]
@justyna-13 can you move all pages to data, for this case you can remove SfHeading from Security page
pages: {
[content: "....", content: [{tabTitle: "...", tabContent: ["paragraph", "paragraph"]}, {}]]
}

**Comment by justyna-13** [comment]
Pages contnet moved to data. Heading and one of the paragraphs've been removed (due to moving content to data). If you prefer contenet of the first paragraph, let me know :)

**Comment by pspaczek** [comment]
@justyna-13 I added some improvement for data

**Pull request merged**